src/tiledb/cloud/geospatial/ingestion.py

```diff
@@ -1402,6 +1412,30 @@ def consolidate_meta(
     logger.info("max memory usage: %.3f GiB", max_memory_usage() / (1 << 30))
 
 
+def remove_dataset_type_from_array_meta(
+    dataset_uri: str,
+    *,
+    verbose: bool = False,
+):
+    """
+    Removes `dataset_type` meta if the ingested result is an array.
+    FIXME: This exists to fix an internal UI issue until formally fixed.
+
+    :param dataset_uri: dataset URI
+    :param verbose: verbose logging, defaults to False
+    """
+    logger = get_logger_wrapper(verbose)
+    if tiledb.object_type(dataset_uri) == "array":
+        logger.info(
+            "Ingested result is an array. " "Removing dataset_type from metadata..."
```

Comment:
What would you think about some structured data at the end of the log message?

```suggestion
            "Removing non-standard dataset_type from ingested array metadata: dataset_uri=%r", dataset_uri
```